### ISSUE TITLE
[Suggestion] - Add more Goodbye options and pronouns and gender

### ISSUE THREAD

**Issue by Jinxred** [Monika-After-Story/MonikaModDev#7442]
Adding things like I'm leaving to see my family and I'm going to the restroom. Also maybe more specific things like I'm leaving to watch a movie. I'm leaving to draw or paint, I'm leaving to practice my instrument. 
Maybe having her reacting to the music you add. 
Add different pronoun options also. Gender doesn't equal pronouns and so just saying that you're a girl or a boy doesn't say what your pronouns are. You can have a what is your gender identity along with a pronoun question but there are more pronouns than what is there. 
For the pronouns maybe make it available to have more than one option selected so she could use things such as they/he or she/it pronouns. Also having an input for neo-pronouns would be nice. Putting a certain pronoun before another could also mean which one they like the most. I don't code so I'm not sure how difficult this would be but perhaps you could have a first selection being like 
**select your first preferred pronouns** and have like she/; he/; they; it/; other(allow them to insert their own text)/;
and then a second section similar like
**select your second preferred pronouns** and have /her; /him; /them; /its; /other(allow them to insert their own text);

Hopefully this is helpful! I love this game!

**Comment by xXMidnight** [comment]
Technically for you to have her react to the music you added would need a database of all the songs in the world, genre, and then a separate line for each song. That's very tedious.

I highly recommend only specific songs.

**Comment by Jinxred** [comment]
yeah, I understand that would be really difficult but maybe she could just react to like songs she suggests? Cause she mentioned the song hibikase but when I added it into the bgm she didn't react at all. It be cool if there were just some little things like that.

**Comment by EclipseTheNerd** [comment]
Not bad ideas. If they got implemented it would be quite nice. As xXMidnight said, it's a very tedious process. I don't code, but even I know _that_ is difficult. Otherwise the pronoun ideas are a really good idea, as usually there are people who prefer they/them or maybe they use neopronouns.

**Comment by ThePotatoGuy** [comment]
for gender pronoun issues, move discussion to #6084

**Comment by fromulus** [comment]
One problem with songs is that the best (and basically only) way of recognising songs is through their filename.

This means you'd have to make sure the song name is spelled out correctly for her to recognise it, and more problematic imo, that you can easily fool her by taking one song and renaming it to another song's name that she's able to react to.

**Comment by Jinxred** [comment]
I mean it would be problematic for that to happen but I feel like it would be easier to just add the song than to change the name of a previous song. Even if someone did change the file name, what would the downsides of that be? Sure they are tricking her into thinking its a different song and I don't agree with doing that, but doing that doesn't have any major damages from my knowledge. I would like to hear your thoughts on this although.

**Comment by ThePotatoGuy** [comment]
song names can be determined by metadata as well -  [see here](https://github.com/Monika-After-Story/MonikaModDev/wiki/Adding-Custom-Music#song-names)

That being said, we used to have reactions for every song that was included in the mod, but dropped it since people started wanting reactions to all music. atm we are not looking to add reactions to music.
